src/app/components/toolbar-panel/toolbar-panel.component.less

```diff
@@ -5,21 +5,101 @@
   padding-left: 20px;
   border-bottom: 1px solid;
   border-bottom-color: @color-pf-black-300;
-  .btn-move { margin-top: 6px; }
-  .toolbar-actions,
-  .toolbar-apf-filter {
-    border-right: 0 !important; /* no move menu yet */
+
+  /*
+    TODO: on the entry "WorkItem Type" of the filter dropdown, there is a
+    border on the right side of the key button missing. The border is actually there,
+    but gets overlayed with the right hand value dropdown. Having fewer text
+    in the button solves the problem. Setting margin to 0px also solves the problem,
+    but then there is a 1px gap in other value dropdowns.
+  */
+
+  .btn {
+    /* font size for all buttons in the toolbar */
+    /* TODO: this should be patternfly default, but it gets overridden somehow */
+    font-size: 12px;
```

Comment:
you define (with comment) that this is a 12 pixels font, yet the site is 14 pixel default...?

Reply:
Mmmh..the original CSS was a mixup between 12px and 14px. Looking at the Patternfly demo, the toolbar uses 12px and it also looks way better than 14px.